charters/DocumentationCharter.md

```diff
@@ -0,0 +1,54 @@
+# Docs Working Group Charter (Proposed)
+
+## Mission and Goals
+
+The Docs Working Group serves as a support, helper, and advisory body to Jupyter subprojects on all aspects of documentation. The core pillars of our mission:
```

Comment:
I love these pillars :-) We have needed this in the community for a long time, really appreciate everyone who is stepping up to help.